.github/workflows/promote-plugin-scope.yml

```diff
@@ -0,0 +1,123 @@
+# Description:
+# Promote plugin scope to universal.
+# It uses action grafana/plugin-ci-workflows/actions/plugins/publish/promote-plugin
+# to change plugin scope to universal.
+# The following inputs should be provided:
+# - environment: Target environment (dev, ops, prod)
+# - plugin_id: Plugin ID to promote
+# - plugin_version: Plugin version to promote
+#
+
+name: Plugins - Promote to Universal Scope
```

Comment:
I wonder if we want to make this a generic action - like "Set plugin's scope" and make scope and input field which default to universal. wdyt?